src/main/java/edu/ucsb/cs156/dining/entities/DiningCommons.java

```diff
@@ -0,0 +1,30 @@
+package edu.ucsb.cs156.dining.entities;
```

Comment:
Why is this here?  What end user functionality does it support?

There is no issue among the five approved issues that describes creating an entity for a DiningCommons (unless it is an oversight, in which case please point it out.)

This should be removed.